data-prepper-plugins/s3-source/src/test/java/com/amazon/dataprepper/plugins/source/S3ObjectWorkerTest.java

```diff
@@ -99,6 +102,7 @@ void parseS3Object_calls_Codec_parse_on_S3InputStream() throws Exception {
         final ResponseInputStream<GetObjectResponse> objectInputStream = mock(ResponseInputStream.class);
         when(s3Client.getObject(any(GetObjectRequest.class)))
                 .thenReturn(objectInputStream);
+        when(compressionEngine.createInputStream(key, objectInputStream)).thenReturn(objectInputStream);
```

Comment:
Can this line and all the repeated similar lines be put into the `setUp()` method instead?